theme/Style/Style.qml

```diff
@@ -8,15 +8,18 @@ QtObject {
     readonly property int pixelSize: fontMetrics.font.pixelSize
 
     // Colors
-    property color ncBlue:      Theme.wizardHeaderBackgroundColor
-    property color ncTextColor: Theme.wizardHeaderTitleColor
-    property color lightHover:  "#f7f7f7"
-    property color menuBorder:  "#bdbdbd"
+    readonly property color ncBlue:      Theme.wizardHeaderBackgroundColor
+    readonly property color ncTextColor: Theme.systemPalette.windowText
+    readonly property color ncSecondaryTextColor: "#808080"
+    readonly property color ncHeaderTextColor: "white"
+    readonly property color lightHover:  Theme.systemPalette.highlight
+    readonly property color menuBorder:  Systray.darkMode ? Qt.lighter(backgroundColor, 3) : Qt.darker(backgroundColor, 1.5)
+    readonly property color backgroundColor: Theme.systemPalette.base

```

Comment:
sorry but I do not get why you cannot use directly `SystemPalette` qml component to achieve that ?

Reply:
SystemPalette is not a QPalette so we cannot apply it to palette properties, also missing a number of properties in a QPalette. Better to get the system palette as a QPalette

Reply:
OK